backend/ohq/urls.py

```diff
@@ -61,6 +69,16 @@
         CourseStatisticView.as_view(),
         name="course-statistic",
     ),
+    path(
+        "courses/<slug:course_pk>/vector_dbs/create",
+        DocumentCreateView.as_view(),
+        name="document-create",
```

Comment:
I think a more fitting name for this path would be `name="vectordb-create-documents"`